src/main/java/com/catapi/model/Breed.java

```diff
@@ -0,0 +1,34 @@
+package com.catapi.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "breed")
+public class Breed {
```

Comment:
Add @NoArgsConstructor and @AllArgsContrsuctor and remove constructors from the classes

Reply:
Added and removed